### PR TITLE
Use Github Workflows instead of Travis CI for testing

### PR DESCRIPTION
**Pull request by malor**
The commit that sets up Github Workflows must go to the default branch. We can submit this, and then branch off version/v4 from master.

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+jobs:
+  ansible:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Ansible
+        run: |
+          python -m pip install --upgrade wheel
```
Comment (ikalnytskyi) [inline .github/workflows/tests.yaml]: nit: I don't think it's actually needed since you don't plan to build your own wheels. If I'm not mistaken, `pip` installs wheels out of box. It's more common to upgrade `pip` itself as the first step (I dunno why).

File: .github/workflows/tests.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          python -m pip install --upgrade ansible
+
+      - name: Run the all-in-one playbook
+        run: |
+          if test -f ansible/all-in-one/site.yml; then
+            # v5
```
Comment (ikalnytskyi) [inline .github/workflows/tests.yaml]: nit: I think we should just create `version/4` branch from master, and assume we're dealing with `v5` only.

File: .github/workflows/tests.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            ansible-playbook --extra-vars '{"require_cgroupsv2": false}' -vv --inventory "xsnippet," --connection=local ansible/all-in-one/site.yml
```
Comment (ikalnytskyi) [inline .github/workflows/tests.yaml]: Q: Why `require_cgroupsv2` are disabled? Because of no intention to use rootless podman?

File: .github/workflows/tests.yaml (hunk 1 above)
Comment (malor) [reply .github/workflows/tests.yaml]: I've copied it from another workflow :) I don't remember _why_ I needed to do that there, though... So maybe it's not required indeed. It's a good point about updating `pip`. Let me do that instead!

File: .github/workflows/tests.yaml (hunk 2 above)
Comment (malor) [reply .github/workflows/tests.yaml]: Yeah, I started wondering that too as soon as I created this PR. Let me update it to cover `version/4` and then create that branch (the first commit that sets up GH Workflows must go to the default branch)

File: .github/workflows/tests.yaml (hunk 3 above)
Comment (malor) [reply .github/workflows/tests.yaml]: I'm not sure if you've seen #33 yet, but configuring cgroupsv2 requires a machine reboot, so I don't think that's going to fly in the GH Workflows virtual environment where we run the Ansible playbook on the same (virtual) host that we configure.

To be honest, I am puzzled by the requirement of cgroupsv2 by rootless Podman: the docs say it _is_ required, but it seems to be working just fine w/o it. The only thing that did not work for me is displaying pod stats (CPU/memory usage, I/O operations, etc).

So, basically, I want to have cgroupsv2 configured in prod, but want to skip it when the playbook is executed in the GH virtual environment.

**Review by ikalnytskyi — APPROVED** [review]
